paasta_tools/cli/cmds/spark_run.py

```diff
@@ -292,6 +292,17 @@ def add_subparser(subparsers):
         default=CLUSTER_MANAGER_K8S,
     )
 
+    list_parser.add_argument(
+        "--enable-dra",
+        help=(
+            "Enable Dynamic Resource Allocation (DRA) for the Spark job as documented in y/spark-dra. DRA aims "
```

Comment:
Concise is better.
Maybe change this to: 
`"Enable Dynamic Resource Allocation (DRA) for the Spark job (y/spark-dra). DRA aims "
            "dynamically scales up and down the instance count based on the number of pending tasks and requirements. Disabled by default. Does not override Spark DRA configs specified by the user."`